.github/workflows/dev-build.yaml

```
@@ -85,12 +95,21 @@ jobs:
           VERSION="dev-$(date "+%Y%m%d-%s")-$(echo "${{ env.COMMIT }}" | cut -c1-8)"
           echo "VERSION=${VERSION}" >> $GITHUB_ENV
 
+      - name: Determine the platforms to build
+        shell: bash
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:5:34: Double quote to prevent globbing and word splitting

(shellcheck)